frontend/src/views/camp/Invitation.vue

```diff
@@ -1,66 +1,94 @@
 <template>
-  <auth-container>
-    <div v-if="!ready" class="text-center">
-      <v-progress-circular size="64" indeterminate color="primary" />
-    </div>
-    <div v-else-if="invitationFound === true">
-      <h1 class="display-1">
-        {{ $tc('views.camp.invitation.title', 0, { campName: invitation().campTitle }) }}
+  <div class="bg">
+    <div class="hill">
+      <v-icon :size="$vuetify.breakpoint.smAndUp ? 300 : 200" class="tent"
+        >$vuetify.icons.tentDay
+      </v-icon>
+      <img src="../../assets/invitation/tree-left.svg" alt="" class="tree l h-full" />
+      <img src="../../assets/invitation/tree-right.svg" alt="" class="tree r h-full" />
+      <div class="localnav justify-space-between d-flex col">
+        <ButtonBack v-if="!$vuetify.breakpoint.mdAndUp" text dark visible-label />
+        <UserMeta v-if="!$vuetify.breakpoint.mdAndUp" just-avatar />
+      </div>
+      <h1 class="wood text-center align-self-end">
+        <span class="rope"></span>
+        <span class="rope l"></span>
+        <span class="subtitle-2 font-weight-bold">{{
+          $tc('views.camp.invitation.title')
+        }}</span>
+        <v-skeleton-loader
+          v-if="invitation()._meta.loading"
+          type="image"
+          width="200"
+          height="48"
+        />
+        <span v-else>{{ invitation().campTitle }}</span>
       </h1>
-
-      <v-spacer />
-      <div v-if="authUser">
-        <v-btn
-          v-if="!invitation().userAlreadyInCamp"
-          color="primary"
-          x-large
-          class="my-4"
-          block
-          @click="acceptInvitation"
-        >
-          {{ $tc('views.camp.invitation.acceptCurrentAuth') }}<br />
-        </v-btn>
-        <div v-else>
-          <v-alert type="warning">
-            {{ $tc('views.camp.invitation.userAlreadyInCamp') }}
+      <div
+        v-if="!invitation()._meta.loading"
+        class="invitation-actions flex-column d-grid gap-2 align-self-start justify-items-center mx-2"
+      >
+        <div v-if="authUser">
+          <v-alert
+            v-if="invitation().userAlreadyInCamp"
+            border="left"
+            colored-border
+            color="primary"
+            :icon="$vuetify.breakpoint.smAndUp ? 'mdi-information-outline' : false"
+            type="info"
+          >
+            {{ $tc('views.camp.invitation.userAlreadyInCamp') }}<br />
+            <div class="mt-2 d-flex flex-wrap gap-2">
+              <v-btn small color="primary" :to="campLink" elevation="0">
+                {{ $tc('views.camp.invitation.openCamp') }}
+              </v-btn>
+              <v-btn
+                color="primary"
+                text
+                small
```

Comment:
Is this small by purpose by purpose? Looks to me strange, that the text for this button is smaller than the ones below (e.g. for "Deny invitation")